js/modules/k6/http/response.go

```diff
@@ -22,102 +22,41 @@ package http
 
 import (
 	"context"
-	"crypto/tls"
-	"encoding/json"
 	"errors"
 	"fmt"
 	"net/url"
 	"strings"
 
-	"github.com/tidwall/gjson"
-
 	"github.com/dop251/goja"
 	"github.com/loadimpact/k6/js/common"
 	"github.com/loadimpact/k6/js/modules/k6/html"
-	"github.com/loadimpact/k6/lib/netext"
+	"github.com/loadimpact/k6/lib/netext/httpext"
 )
 
-// ResponseTimings is a struct to put all timings for a given HTTP response/request
-type ResponseTimings struct {
-	Duration       float64 `json:"duration"`
-	Blocked        float64 `json:"blocked"`
-	LookingUp      float64 `json:"looking_up"`
-	Connecting     float64 `json:"connecting"`
-	TLSHandshaking float64 `json:"tls_handshaking"`
-	Sending        float64 `json:"sending"`
-	Waiting        float64 `json:"waiting"`
-	Receiving      float64 `json:"receiving"`
-}
-
 // Response is a representation of an HTTP response to be returned to the goja VM
-type Response struct {
-	ctx context.Context
+// TODO: refactor after https://github.com/dop251/goja/issues/84
```

Comment:
Definitely not in this PR - as it will mean upgrading goja in this one as well and this one already has a bunch of non related things in it. I added https://github.com/loadimpact/k6/issues/946

Reply:
:+1: